Documentation/Storage-Configuration/Advanced/ceph-configuration.md

```diff
@@ -47,12 +47,24 @@ kubectl apply -f common.yaml -f operator.yaml -f cluster.yaml # add other files
 
 ## Deploying a second cluster
 
-If you wish to create a new CephCluster in a different namespace than `rook-ceph` while using a single operator to manage both clusters execute the following:
+If you wish to create a new CephCluster in a different namespace than `rook-ceph` while using a
+single operator to manage both clusters change `SECOND_ROOK_CLUSTER_NAMESPACE`. The `common-second-cluster`
+manifests is commented to allow for easy `sed` replacements.
 
 ```console
 cd deploy/examples
+export SECOND_ROOK_CLUSTER_NAMESPACE="rook-ceph"
 
-NAMESPACE=rook-ceph-secondary envsubst < common-second-cluster.yaml | kubectl create -f -
+sed -i.bak \
```

Comment:
This is a very complex sed statement. What if we simplified it by the following?
1. In the yaml, get rid of the env var and just define an example namespace directly: `namespace: rook-ceph-secondary # namespace:cluster`
2. The command to set the desired namespace would just be a simple replacement of all `rook-ceph-secondary` and operator namespace with the desired namespaces
3. The CI would no longer need to substitute the var like this: `NAMESPACE=rook-ceph-secondary envsubst <common-second-cluster.yaml`

Reply:
@travisn do I need to adjust some CI for this?

Reply:
On [this line](https://github.com/rook/rook/blob/master/tests/scripts/github-action-helper.sh#L414) there is no need for a substitution since the example is already using the expected namespaces by default. They only need to be replaced if the user wants a different namespace.